test/controllers/tasks_controller_test.rb

```diff
@@ -48,8 +48,8 @@ def in_taxonomy_scope(organization, location = nil)
             @locations = [0, 0].map { FactoryBot.create(:location) }
             @tasks = [0, 0].map { FactoryBot.create(:some_task) }
             @tasks.zip(@organizations, @locations).each do |task, org, loc|
-              Lock.link!(org, task.id)
-              Lock.link!(loc, task.id)
+              Link.link!(org, task)
```

Comment:
Should we maybe provide a compatibility layer which would contain `Lock.link!` and would print a deprecation warning?

Reply:
sounds reasonable to me